### PR TITLE
Added required messages

### PR DESCRIPTION
**Pull request by PJSosnowski**
Added messages required for 
motech/motech-ui#81

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by jredlarski** [comment]
test this please

File: platform/email/src/main/resources/webapp/messages/messages.properties [hunk 1]
```diff
@@ -77,3 +77,5 @@ email.logging.creationDate=Creation Date
 email.logging.modifiedDate=Modified Date
 email.logging.deliveryTime=Delivery Time
 email.logging.deliveryStatus=Status
+
+email.emptyLogsList=No logs found
```
Comment (jredlarski) [inline platform/email/src/main/resources/webapp/messages/messages.properties]: @PJSosnowski Change this to `email.logging.emptyLogsList`

File: modules/admin/src/main/resources/webapp/messages/messages.properties [hunk 2]
```diff
@@ -259,4 +258,8 @@ admin.taskAction.actionName=Status message
 admin.taskAction.text=Text
 admin.taskAction.moduleName=Module name
 admin.taskAction.level=Level
-admin.taskAction.timeout=Timeout
\ No newline at end of file
+admin.taskAction.timeout=Timeout
+
+admin.emptyRecipientList=No recipients will be added
```
Comment (jredlarski) [inline modules/admin/src/main/resources/webapp/messages/messages.properties]: Change to `admin.messages.emptyRecipientList`

File: modules/admin/src/main/resources/webapp/messages/messages.properties [hunk 3]
```diff
@@ -259,4 +258,8 @@ admin.taskAction.actionName=Status message
...
+admin.emptyBundlesList=No bundles found
```
Comment (jredlarski) [inline modules/admin/src/main/resources/webapp/messages/messages.properties]: Change to `admin.bundles.emptyBundlesList`

File: modules/admin/src/main/resources/webapp/messages/messages.properties [hunk 4]
```diff
@@ -259,4 +258,8 @@ admin.taskAction.actionName=Status message
...
+admin.emptyMessagesList=No messages found
```
Comment (jredlarski) [inline modules/admin/src/main/resources/webapp/messages/messages.properties]: Change to `admin.messages.emptyMessagesList`

**Comment by jredlarski** [comment]
test this please

**Comment by jredlarski** [comment]
test this please

**Comment by jredlarski** [comment]
test this please

**Pull request merged**